### PR TITLE
Add column description for the Customer Segment page

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request adds column descriptions for the Customer Segment page.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/marketing/customer-segment-create.html

### CODE REVIEW

File: src/marketing/customer-segment-create.md [hunk 1]
```diff
@@ -11,6 +11,13 @@ _Customer Segments list_
 
 The following example shows how to create a customer segment that targets female customers in Los Angeles.
 
+|Column|Description|
+|--- |--- |
+|ID|A unique, sequential number that is assigned when a new segment is saved for the first time.|
+|Segment|The name of the segment.|
+|Status|The current segment status. Options include: Active / Inactive.|
+|Website|The website where the segment applies.|
+
```
Comment (leslietilling) [inline src/marketing/customer-segment-create.md:20]: This is disruptive in this location and not really needed. Let's just keep the lists to the subsections below.

```suggestion
```

**Review by leslietilling — CHANGES_REQUESTED** [review]

File: src/marketing/customer-segment-create.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/marketing/customer-segment-create.md]: Thank you for your advice! Done ✅ 
@leslietilling

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**